src/forms/input/Input.tsx

```diff
@@ -81,6 +84,7 @@ export class Input extends React.PureComponent<InputProps> {
                         <input
                             type={type || "text"}
                             id={name}
+                            value={value}
```

Comment:
Can we add story with new 'value' prop in storybook?

Reply:
Addressed...